specification/Metadata/0.0.0/README.md

```diff
@@ -74,50 +138,75 @@ Using terminology from the geospatial industry, **features** are geometric entit
 
 This specification adds a mechanism for storing metadata in existing 3D model formats such as Khronos Group's glTF or Cesium's 3D Tiles. Metadata can be applied at various granularities: per-vertex, per-texel, as well as larger structures from meshes to tilesets. This metadata can be used for analytics purposes, as well as styling in a 3D renderer.
 
-Guiding principles for this specification include:
+Guiding principles include:
 
 - Design a data format that allows for runtime efficiency, even for large datasets.
-- Keep class definitions separate from instantiation. This allows for greater flexibility of data formats, and allows for reuse of metadata definitions.
+- Keep class definitions separate from instantiation. This allows for greater flexibility of data formats, and allows for reuse of metadata definitions.s
+
+For a complete list of vocabulary used, see the [Glossary](#glossary)
 
 ## Concepts
 
-### Classes
+### Data Types
 
-A **class** describes a collection of related pieces of metadata called **properties** and their respective data types. In some ways, this is similar to describing the columns of a database table. However, this analogy is not completely applicable, as this specification allows for texture storage, not just columnar formats.
+**Primitive types** are data types that cannot be divided further. These include:
 
-A class definition describes what metadata is available. However, it does not describe how this metadata is stored. The section on [Instances](#instances) will provide more information.
+* Boolean values (`BOOLEAN`)
+* Signed integers (`INT8`, `INT16`, `INT32`, `INT64`)
+* Unsigned integers (`INT8`, `INT16`, `INT32`, `INT64`)
+* Floating point numbers (`FLOAT16`, `FLOAT32`, `FLOAT64`)
+* Strings (`STRING`)
 
-The following example shows the basics of how classes describe the data types, without describing where the data is stored. The following section will show how to connect these classes to the actual metadata in various storage formats.
+Even though a string is a sequence of characters, it is considered to be a single indivisible value when storing metadata.
 
+**Array types** (`ARRAY`) are sequences of a single primitive type. The values are stored contiguously. Arrays can be fixed length or variable length.
```

Comment:
To be more consistent, do we want to use "variable length" or "variable size"?

In the rest of the document, it seems like "variable-size array" is far more common. Maybe we should go with that?